build.gradle

```diff
@@ -246,6 +246,7 @@ task fastbuild(dependsOn: [
 
 task travisbuild(dependsOn: [
     fastbuild,
+    ':dbfit-java:db2:integrationTest',
```

Comment:
What about moving that to the end of the list? I think mysql is the most light-weight so good to have it on first place I think.